servers/rendering_server.cpp

```diff
@@ -3016,7 +3016,7 @@ void RenderingServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("environment_set_bg_color", "env", "color"), &RenderingServer::environment_set_bg_color);
 	ClassDB::bind_method(D_METHOD("environment_set_bg_energy", "env", "multiplier", "exposure_value"), &RenderingServer::environment_set_bg_energy);
 	ClassDB::bind_method(D_METHOD("environment_set_canvas_max_layer", "env", "max_layer"), &RenderingServer::environment_set_canvas_max_layer);
-	ClassDB::bind_method(D_METHOD("environment_set_ambient_light", "env", "color", "ambient", "energy", "sky_contibution", "reflection_source"), &RenderingServer::environment_set_ambient_light, DEFVAL(RS::ENV_AMBIENT_SOURCE_BG), DEFVAL(1.0), DEFVAL(0.0), DEFVAL(RS::ENV_REFLECTION_SOURCE_BG));
+	ClassDB::bind_method(D_METHOD("environment_set_ambient_light", "env", "color", "ambient", "energy", "sky_contribution", "reflection_source"), &RenderingServer::environment_set_ambient_light, DEFVAL(RS::ENV_AMBIENT_SOURCE_BG), DEFVAL(1.0), DEFVAL(0.0), DEFVAL(RS::ENV_REFLECTION_SOURCE_BG));
```

Comment:
This would need to be fixed in the docs too, which you can do by running `doctool`.

Reply:
Mmm, now that I think of it, would this be an API-breaking change too (like the enum changes above that I removed)? (Does GDScript have keyword arguments, i.e. can there be user code out there that does `environment_set_ambient_light(..., sky_contibution=123)`?)

Reply:
It's not breaking compat for GDScript which doesn't support named arguments.

It does break compat for C#, but so far we've considered this outside the scope of the API compatibility we can guarantee. We wouldn't do such rename in a patch release but in a minor version this should be fine.